size_comparison/.cargo/config

```diff
@@ -0,0 +1,8 @@
+[build]
+target = "thumbv7m-none-eabi"
```

Comment:
Isn't the majority of boards we're supporting cortex-m4?

Reply:
Perhaps, but thumbv7m-none-eabi is what I have available. The particular architecture doesn't really matter here.

Reply:
I see